app/controllers/invoice_lists_controller.rb

```diff
@@ -146,4 +155,17 @@ def recipient_ids_from_people_filter
   def authorize_class
     authorize!(:index_invoices, parent)
   end
+
+  def permitted_params
```

Comment:
dann packt man das hier noch in die permitted attrs. 
jetzt frage ich mich einmal mehr ... gibt es eine elegantere lösung anstatt den sti name als parameter mitzugeben?